packages/omang-service-mediator/docker-compose.dev.yml

```diff
@@ -4,7 +4,7 @@ services:
   oracle-db:
     image: ${OMANG_ORACLE_DOCKER_IMAGE}
     environment:
-      - ORACLE_SID=${OMANG_ORACLE_SID}
+      - ORACLE_SID=${OMANG_ORACLE_DB_SID}
```

Comment:
in the env file the variable name is OMANG_ORACLE_SID

Reply:
I have updated the variable as well. I made changes just to standardize the variables naming